### ISSUE TITLE
Have public command show SG's that are allowing the exposure

### ISSUE THREAD

**Issue by 0xdabbad00** [duo-labs/cloudmapper#233]
A question that quickly will arise upon finding a resource is exposed, is how is it exposed?  The public command should show what Security Groups are allowing the exposure.

**Comment by 0xdabbad00** [comment]
Currently, the `public` command for the demo account shows:
```
$ python cloudmapper.py public --account demo
{
    "account": "demo",
    "arn": "arn:aws:ec2:us-east-1:123456789012:instance/weblb/subnet-00000001",
    "hostname": "weblb.us-west-2.elb.amazonaws.com",
    "ports": "443",
    "public_sgs": [
        "sg-00000003"
    ],
    "type": "elb"
}
{
    "account": "demo",
    "arn": "arn:aws:ec2:us-east-1:123456789012:instance/weblb/subnet-00000002",
    "hostname": "weblb.us-west-2.elb.amazonaws.com",
    "ports": "443",
    "public_sgs": [
        "sg-00000003"
    ],
    "type": "elb"
}
{
    "account": "demo",
    "arn": "arn:aws:ec2:us-east-1:123456789012:instance/webalb/subnet-00000002",
    "hostname": "webalb",
    "ports": "443",
    "public_sgs": [
        "sg-00000003"
    ],
    "type": "elb"
}
```
This could be further improved by giving more info on the SG's, like the name, and the ports that each SG is allowing open.

**Comment by 0xdabbad00** [comment]
Resolved in #351
